test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/BankAccountControllerSpec.scala

```diff
@@ -111,6 +131,49 @@ class BankAccountControllerSpec
 
   "Bank Account Controller" when {
 
+    "Check Bank Account Details" should {
+
+      "Redirect when MaskedBankDetails is empty" in {
+        val maskedBankDetails = MaskedBankDetails(None, None)
+        val (session, _, _)   = sessionWithMaskedBankDetails(Some(maskedBankDetails))
+        inSequence {
+          mockAuthWithNoRetrievals()
+          mockGetSession(session)
+        }
+        val request           = FakeRequest()
+        val result            = controller.checkBankAccountDetails()(request)
+        checkIsRedirect(result, routes.BankAccountController.enterBankAccountDetails())
+      }
+
+      "Ok when MaskedBankDetails has consigneeBankDetails" in {
+        val consigneeDetails  = sample[ConsigneeBankDetails]
+        val maskedBankDetails = MaskedBankDetails(Some(consigneeDetails), None)
+        val (session, _, _)   = sessionWithMaskedBankDetails(Some(maskedBankDetails))
+        inSequence {
+          mockAuthWithNoRetrievals()
+          mockGetSession(session)
+        }
+        val request           = FakeRequest()
+        val result            = controller.checkBankAccountDetails()(request)
+        status(result) shouldBe OK
+        //checkPageIsDisplayed(result, messageFromMessageKey("bank-details.title"))
```

Comment:
your doing a redirect for this and there's a helper method in controller spec that helps you check for this - use that

Reply:
This is not a redirect!
As you can see i'm expecting an OK answer.
This is testing the other cases (not related to the issue) just added them, since it was just 2 lines apart in the code.